server/test/services/export/JsonExporterTest.java

```diff
@@ -178,28 +503,60 @@ void assertValueAtPath(String path, String value) {
       assertThat(resultJson.readString(Path.create(path)).get()).isEqualTo(value);
     }
 
-    void assertValueAtPath(int resultNumber, String innerPath, String value) {
+    void assertValueAtPath(String path, Long value) {
```

Comment:
I **think** all these methods can be private, as they'd still be visible to the outer class.

Reply:
done.